tokenmagic/module/autoTemplate/dnd5e.js

```diff
@@ -0,0 +1,182 @@
+import { defaultOpacity, emptyPreset } from '../constants.js';
+
+export class AutoTemplateDND5E {
+	static get defaultConfiguration() {
+		const defaultConfig = {
```

Comment:
I'm open to alternative/additional defaults for all these templates, I just knocked these up quickly to give users some out-of-the-box effects.

Reply:
The default templates suit me well! I just need to rework the thick fog shader, which is too compute-heavy for mid-range gpu.

Reply:
I'd love to add some more defaults for spells that don't have direct damage, but there are... quite a lot of spells to review :wink: 

The other thing I'm thinking about is that once a user saves the template settings into their world, any changes that we make to these defaults won't be reflected. I guess we could use the migration mechanism and update settings if they match the previous defaults, if we need to push some sort of critical change to the default settings. Maybe we don't need to worry too much about this though.

Reply:
In theory, an evolution of the data in the presets should not alter its visual qualities (except for important reasons). Indeed, if we need to make updates, the migration mechanism is there for that.
And yeah, there is a lot of spells ! 😉

Reply:
If you want to start a defaults spell effects project, we can share the task. 😉